extensions/amp-carousel/amp-carousel-analytics.md

```diff
@@ -63,4 +63,4 @@ The `amp-carousel-next` event is issued when there is a travesal to the previous
 
 ### Vars
 
-Description of the variables can be found in the [analytics-vars.md](/extensions/amp-analytics/analytics-vars.md#fromslide) file.
+Description of the variables can be found in the [analytics-vars.md](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md#fromslide) file.
```

Comment:
I'm sorry, I presented the wrong fix in my comment on the other PR. This was a genuine bad relative link, and the simpler fix is to make the link `../amp-analytics/analytics-vars.md#fromslide`.